src/Arcus.WebApi.Security/Authentication/ConfigurationValidationLocation.cs

```diff
@@ -30,7 +30,7 @@ internal class ConfigurationValidationLocation : IX509ValidationLocation
         /// </summary>
         /// <param name="configurationKey">The configured key for which the expected certificate value is registered.</param>
         /// <param name="services">The services collections of the HTTP request pipeline to retrieve registered instances.</param>
-        public Task<string> GetExpectedCertificateValueForConfiguredKey(string configurationKey, IServiceProvider services)
+        public Task<string> GetExpectedCertificateValueForConfiguredKeyAsync(string configurationKey, IServiceProvider services)
```

Comment:
:warning: Careful, should not be breaking but additional

Reply:
Yes, that was the question I had in the mail I send you: this was completed yesterday, is it safe not assume that no new versions are pushed since then?